*** /usr/share/lightdm-gtk-greeter/greeter.ui.orig	2012-11-20 22:01:22.000000000 +0000
--- /usr/share/lightdm-gtk-greeter/greeter.ui	2014-05-15 04:40:30.863671766 +0000
***************
*** 31,37 ****
                  <property name="visible">True</property>
                  <property name="can_focus">False</property>
                  <property name="pixel_size">64</property>
!                 <property name="icon_name">computer</property>
                </object>
                <packing>
                  <property name="expand">False</property>
--- 31,37 ----
                  <property name="visible">True</property>
                  <property name="can_focus">False</property>
                  <property name="pixel_size">64</property>
!                 <property name="icon_name">dax</property>
                </object>
                <packing>
                  <property name="expand">False</property>
